oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/token/OAuth2TokenMetadata.java

```diff
@@ -33,13 +33,18 @@
  * @see OAuth2Tokens
  */
 public class OAuth2TokenMetadata implements Serializable {
+
 	private static final long serialVersionUID = Version.SERIAL_VERSION_UID;
 	protected static final String TOKEN_METADATA_BASE = "metadata.token.";
 
 	/**
 	 * The name of the metadata that indicates if the token has been invalidated.
 	 */
 	public static final String INVALIDATED = TOKEN_METADATA_BASE.concat("invalidated");
+	public static final String JWT = TOKEN_METADATA_BASE.concat("jwt");
+	public static final String AUTHORIZED_SCOPES = TOKEN_METADATA_BASE.concat("authorized-scopes");
+
+	public static final OAuth2TokenMetadata EMPTY = OAuth2TokenMetadata.builder().build();
```

Comment:
Please remove as each `OAuth2Token` should have it's own instance of `OAuth2TokenMetadata`